### DISCUSSION TITLE
Add a confirmation dialog when drag and dropping a folder

### DISCUSSION THREAD

**Discussion by venilark** [godotengine/godot-proposals#6724] [General Discussions]
A few minutes ago I did this by mistake, the project crashed and then I had lots of broken dependencies issues so I had to use a backup project. This wouldn't prevent dependency issues but at least you'll be aware if you are moving the folder by mistake.

**Comment by Zireael07** [comment]
I believe it was already suggested, although I can't find the request # atm
